lib/cog/chat/message.ex

```diff
@@ -7,6 +7,7 @@ defmodule Cog.Chat.Message do
   field :user, [object: Cog.Chat.User], required: true
   field :text, :string, required: true
   field :provider, :string, required: true
+  field :thread_id, :string, required: false
```

Comment:
Similar to above I'm thinking we should replace `:thread_id` with a `metadata` module and field like so:

`field :metadata, [object: Cog.Chat.MessageMetadata], required: false`